kuma/javascript/src/payments/components/useScriptLoading.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import { useEffect, useState } from 'react';
+
+/**
+ * Loads the script given by the URL and cleans up after itself
+ * @param string url of the script to load
```

Comment:
`string` should be `{string}`
https://jsdoc.app/tags-param.html

Reply:
Gawd I'm not going to miss these sporadic jsdoc comments when we move the Yari front-end to TS. BUT, I often like the doc string comment that sometimes explains why the function exists.